ui/inspectors/calendar-task.reel/rsync-args.reel/rsync-args.js

```diff
@@ -32,6 +32,7 @@ exports.RsyncArgs = AbstractInspector.specialize({
                 this.object.clear();
                 this.object.push({rsync_properties: {}});
                 this.object.__type = this.type;
+                this.object[0].remote_ssh_port = 22;
```

Comment:
no need to do this erin, as I have `params.get('remote_ssh_port', 22)` in the middleware already.

Other than that it looks ok. did not get time to test it though